services/aster-nix/src/fs/utils/random_test.rs

```diff
@@ -0,0 +1,532 @@
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
Can we move this random test until into the root of fs? So we can use it to test other filesystems.